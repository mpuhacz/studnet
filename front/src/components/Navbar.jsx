import React from 'react';

const Navbar = () => (
  <div className="ui fixed inverted menu">
  <div className="ui container">
    <a href="#" className="header item">
      <img className="logo" src="https://semantic-ui.com/examples/assets/images/logo.png" alt=""/>
      studNet
    </a>
    <a href="#" className="item">Strona GÅ‚owna</a>
    <div className="ui simple dropdown item">
      Zmien Uczelnie <i className="dropdown icon"></i>
      <div className="menu">
        <div className="header">Twoje Uczelnie:</div>
        <a className="item" href="#">Politechnika</a>
        <a className="item" href="#">Uniwersytet</a>
      </div>
    </div>
    <div className="ui simple dropdown item">
      Zmien Kierunek <i className="dropdown icon"></i>
      <div className="menu">
        <div className="header">Twoje Kierunki:</div>
        <a className="item" href="#">Informatyka</a>
        <a className="item" href="#">Bioinformatyka</a>
      </div>
    </div>
  </div>
  </div>
);

export default Navbar;
