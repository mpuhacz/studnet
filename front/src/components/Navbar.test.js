import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Navbar from './Navbar';

describe('<Navbar />', () => {
  it('renders Navbar without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Navbar/>, div);
  });

  it('renders Navbar form with correct specification', () => {
    const wrapper = renderer.create(<Navbar/>);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
