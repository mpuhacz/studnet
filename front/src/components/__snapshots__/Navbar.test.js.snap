exports[`<Navbar /> renders Navbar form with correct specification 1`] = `
<div
  className="ui fixed inverted menu">
  <div
    className="ui container">
    <a
      className="header item"
      href="#">
      <img
        alt=""
        className="logo"
        src="https://semantic-ui.com/examples/assets/images/logo.png" />
      studNet
    </a>
    <a
      className="item"
      href="#">
      Strona GÅ‚owna
    </a>
    <div
      className="ui simple dropdown item">
      Zmien Uczelnie 
      <i
        className="dropdown icon" />
      <div
        className="menu">
        <div
          className="header">
          Twoje Uczelnie:
        </div>
        <a
          className="item"
          href="#">
          Politechnika
        </a>
        <a
          className="item"
          href="#">
          Uniwersytet
        </a>
      </div>
    </div>
    <div
      className="ui simple dropdown item">
      Zmien Kierunek 
      <i
        className="dropdown icon" />
      <div
        className="menu">
        <div
          className="header">
          Twoje Kierunki:
        </div>
        <a
          className="item"
          href="#">
          Informatyka
        </a>
        <a
          className="item"
          href="#">
          Bioinformatyka
        </a>
      </div>
    </div>
  </div>
</div>
`;
